### PR TITLE
Add the initial version of the MTMC demo

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

**Comment by IRDonch** [comment]
@sovrasov Please retarget this to the `develop` branch.

File: demos/python_demos/multi_camera_multi_person_tracking/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+This demo demonstrates how to run Multi Camera Multi Person demo using OpenVINO<sup>TM</sup>
+
+## Setup
+
+### Prerequisites
+
+* Ubuntu 16.04
+* Python 3.5.2
+* OpenVINO 2019 R2 with Python API
```
Comment (IRDonch) [inline demos/python_demos/multi_camera_multi_person_tracking/README.md]: I don't think this needs to be mentioned. The demos are installed as part of the OpenVINO toolkit; there is no way for these prerequisites to be unfulfilled.

File: demos/python_demos/multi_camera_multi_person_tracking/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### Installation
+
+To install required dependencies run
+
+```bash
+$ cat requirements.txt | xargs -n 1 -L 1 pip3 install
```
Comment (IRDonch) [inline demos/python_demos/multi_camera_multi_person_tracking/README.md]: Why not just `pip3 install -r requirements.txt`?

File: demos/python_demos/multi_camera_multi_person_tracking/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+1. The demo expects the next models in the Intermediate Representation (IR) format:
+
+   * Person detection model
+   * Person re-identification model
+
+It can be your own models or pre-trained model from OpenVINO Open Model Zoo.
+In the `models.lst` are the list of appropriate models for this demo 
+that can be obtained via `Model downloader`. 
+Please see more information about `Model downloader` [here](https://github.com/opencv/open_model_zoo/blob/master/tools/downloader/README.md).
```
Comment (IRDonch) [inline demos/python_demos/multi_camera_multi_person_tracking/README.md]: This should be a relative link.

File: demos/python_demos/multi_camera_multi_person_tracking/run.py [hunk 4]
```diff
@@ -0,0 +1,146 @@
+"""
```
Comment (IRDonch) [inline demos/python_demos/multi_camera_multi_person_tracking/run.py]: The main script of the demo should have the same name as the demo itself, for consistency with other demos.

**Comment by sovrasov** [comment]
@IRDonch all the comments are resolved in #343

**Pull request closed without merging**